src/app/components/AmpExperiment/index.tsx

```diff
@@ -0,0 +1,50 @@
+/** @jsx jsx */
+/* @jsxFrag React.Fragment */
+import { jsx } from '@emotion/react';
+import React from 'react';
+import { Helmet } from 'react-helmet';
+
+type Variant = string;
+type Experiment = string;
+type trafficAllocationPercentage = number;
+
+type AmpExperimentConfig = {
+  [key: Experiment]: {
+    sticky?: boolean;
+    consentNotificationId?: string;
+    variants: {
+      [key: Variant]: trafficAllocationPercentage;
```

Comment:
Tiny typo (capitalisation).

```suggestion
type TrafficAllocationPercentage = number;

type AmpExperimentConfig = {
  [key: Experiment]: {
    sticky?: boolean;
    consentNotificationId?: string;
    variants: {
      [key: Variant]: TrafficAllocationPercentage;
```